Lib/multiprocessing/popen_forkserver.py

```diff
@@ -50,6 +50,7 @@ def _launch(self, process_obj):
 
         self.sentinel, w = forkserver.connect_to_new_process(self._fds)
         self.finalizer = util.Finalize(self, os.close, (self.sentinel,))
+        parent_sentinel = os.dup(w)
```

Comment:
This variable isn't used anywhere. Is it a bug?

Reply:
It leaves the write end opened as  `w`is closed at the end of the following context manager  (upon your request). But now that I fully understood https://github.com/python/cpython/pull/13247#discussion_r285666739, I guess it is better to have it as a private attribute of the process object and register a Finalizer.